update_tag_sample_page.py

```diff
@@ -0,0 +1,50 @@
+import sys
+import argparse
+from rest import McM
+sys.path.append('/afs/cern.ch/cms/PPD/PdmV/tools/McM/')
+
+parser = argparse.ArgumentParser(description='Tag samples for an analysis subgroup')
+parser.add_argument('--input_file', default='text_not_existing.txt', required=True,
+                    help='input file where miniAOD names are stored')
+parser.add_argument('--tag', default='default_tag', required=True,
```

Comment:
I would not use `default=` here, otherwise half of McM will be tagged with `default_tag` which is useless.